crate/client/src/config.rs

```diff
@@ -15,17 +16,21 @@ use x509_cert::Certificate as X509Certificate;
 #[cfg(target_os = "linux")]
 use crate::client_bail;
 use crate::{
-    error::{result::RestClientResultHelper, ClientError},
+    error::{
+        result::{ClientResult, RestClientResultHelper},
+        ClientError,
+    },
     FindexClient,
 };
 
 /// Returns the path to the current user's home folder.
 ///
-/// On Linux and macOS, the home folder is typically located at `/home/<username>`
-/// or `/Users/<username>`, respectively. On Windows, the home folder is typically
-/// located at `C:\Users\<username>`. However, the location of the home folder can
-/// be changed by the user or by system administrators, so it's important to check
-/// for the existence of the appropriate environment variables.
+/// On Linux and macOS, the home folder is typically located at
+/// `/home/<username>` or `/Users/<username>`, respectively. On Windows, the
+/// home folder is typically located at `C:\Users\<username>`. However, the
+/// location of the home folder can be changed by the user or by system
+/// administrators, so it's important to check for the existence of the
+/// appropriate environment variables.
 ///
 /// Returns `None` if the home folder cannot be determined.
 fn get_home_folder() -> Option<PathBuf> {
```

Comment:
Isn't [this function](https://doc.rust-lang.org/std/env/fn.home_dir.html) what we want?

Reply:
[This](https://doc.rust-lang.org/std/env/fn.home_dir.html) has an unexpected behavior on Windows. I believe we should have a Windows version of this CLI.

Reply:
mmh, maybe sending a fix for this unexpected behavior to the upstream could be a good idea if you have some spare time one day :)